demos/demo_solo12.cpp

```diff
@@ -136,21 +129,26 @@ static THREAD_FUNCTION_RETURN_TYPE control_loop(void* thread_data_void_ptr)
         // print -----------------------------------------------------------
         if ((count % 1000) == 0)
         {
+            blmc_robots::Vector12d current_index_to_zero =
+                joint_index_to_zero - robot->get_joint_positions();
+
             printf("\33[H\33[2J");  // clear screen
-            print_vector("sliders_filt", sliders_filt);
-            print_vector("sliders_zero", sliders_zero);
-            print_vector("sliders_raw ", robot->get_slider_positions());
-            print_vector("des_joint_tau", desired_torque);
-            print_vector("    joint_pos", robot->get_joint_positions());
-            print_vector("    joint_vel", robot->get_joint_velocities());
-            print_vector("des_joint_pos", desired_joint_position);
+            print_vector(" sliders_filt", sliders_filt);
+            print_vector(" sliders_zero", sliders_zero);
+            print_vector(" sliders_raw ", robot->get_slider_positions());
+            print_vector(" des_joint_tau", desired_torque);
+            print_vector("     joint_pos", robot->get_joint_positions());
+            print_vector("     joint_vel", robot->get_joint_velocities());
+            print_vector(" des_joint_pos", desired_joint_position);
+            print_vector("zero_joint_pos", current_index_to_zero);
```

Comment:
```
print_vector("zero_joint_pos", current_index_to_zero);
```

This is my new favorite part about the demo: It prints out the number you have to put into the calibration yaml right away :)